requirements/edx/base.txt

```diff
@@ -737,7 +737,7 @@ markdown==3.3.7
     #   openedx-django-wiki
     #   staff-graded-xblock
     #   xblock-poll
-markupsafe==2.1.5
+markupsafe==3.0.1
```

Comment:
`markupsafe==3.0.0` dropped support for `Python 3.8` and added support for `Python 3.13`. [[CHANGELOG](https://github.com/pallets/markupsafe/releases/tag/3.0.0)]
Introduced some breaking changes in the new major release.